index.js

```diff
@@ -0,0 +1,16 @@
+const hamburgerMenu = document.querySelector('.hamburgerMenu');
+const navMenu = document.querySelector('.navMenu');
+const navLink = document.querySelectorAll('.navLink');
+const line = document.querySelector('.hamburgerLine')
+function mobileMenu() {
+  hamburgerMenu.classList.toggle('active');
+  navMenu.classList.toggle('active');
+}
+hamburgerMenu.addEventListener('click', mobileMenu);
```

Comment:
- [ ] When I click on the hamburger, the mobile menu doesn't appear. please ensure that on click of the hamburger, the mobile menu appears as specified in the requirements 👍